src/scripts/sync_subclassof.py

```diff
@@ -0,0 +1,110 @@
+"""Create a robot template for purpose of syncing subclassof relations.
+
+Resources
+- GitHub issue: https://github.com/monarch-initiative/mondo-ingest/issues/92
+- GitHub PR: https://github.com/monarch-initiative/mondo-ingest/pull/363
+"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Any, Dict, List, Tuple, Union
+
+import pandas as pd
+from oaklib import get_adapter
+from oaklib.interfaces.basic_ontology_interface import BasicOntologyInterface, RELATIONSHIP
+from oaklib.types import CURIE, URI
+
+from src.scripts.utils import DEFAULT_PREFIXES_CSV, get_monarch_curies_converter, remove_angle_brackets
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+ONTOLOGY_DIR = SRC_DIR / 'ontology'
+DEFAULT_OUTPATH = ONTOLOGY_DIR / 'reports' / 'sync-subclassof.robot.template.tsv'
+DEFAULT_ONTOLOGY_DB_PATH = ONTOLOGY_DIR / 'tmp' / 'merged.db'
+ROBOT_ROW = [{
+    'term_id': 'ID',
+    'parent_id': 'SC % SPLIT=|',
+    'xref_ids': 'A oboInOwl:hasDbXref SPLIT=|',  # todo: Nico may provide different value
+}]
+
+
+def sync_subclassof(
+    outpath: str = DEFAULT_OUTPATH, prefixes_csv_path: str = DEFAULT_PREFIXES_CSV,
+    ontology_db_path: str = DEFAULT_ONTOLOGY_DB_PATH
+):
+    """Run"""
+    # Vars
+    conv = get_monarch_curies_converter(prefixes_csv_path)
+    compress = lambda x: conv.compress(x) if conv.compress(x) else x
+    oi: BasicOntologyInterface = get_adapter(ontology_db_path)
+
+    # Get all terms
+    ids_all: List[Union[CURIE, URI]] = [x for x in oi.entities(filter_obsoletes=False)]
+    mondo_ids: List[CURIE] = [x for x in ids_all if x.startswith('MONDO:')]
+
+    # todo: Move this further down?
+    # TODO: filter based on this
+    from datetime import datetime
+    t1 = datetime.now()
+
+
+    exclusion_rels: List[Tuple[str, str]] = []
+    i = 0
+    # Note: in 1 example, using include_all_triples=True only added: ['rdf:type', 'rdfs:subClassOf']
+    for rel in oi.entities_metadata_statements(mondo_ids, ['obo:mondo#excluded_subClassOf']):
```

Comment:
## `obo:mondo#excluded_subClassOf`
@matentzn I'm in the progress filtering out these excluded subclass relationships. Note that originally I was going to do this in SPARQL, but I found a way in OAK. There's a possibility it will be much less performant than SPARQL. If so, I can change the implementation and also open a issue in OAK about it.

Reply:
Important thing to sort out. If necessary ping Chris and Harshad directly on slack!

Reply:
This is a non-issue for the scope of this PR now. However, I created an issue in OAK:
- https://github.com/INCATools/ontology-access-kit/issues/676